iphone/Maps/Classes/CustomViews/MapViewControls/Search/Console/MWMConsole.mm

```diff
@@ -15,6 +15,9 @@ + (BOOL)performCommand:(NSString *)cmd
 
   if ([self performSound:cmd])
     return YES;
+  
+  if ([self perform3dMode:cmd])
```

Comment:
We did these commands in applications specially to make normal user experience - when user types command, app hides search panel and navigates back to the map and etc. - users see result of command immediately.
Plus for changing map style, iOS and Android apps do different steps of pre and post actions for EGL - it was not possible to make map style change only in the core.